eng/scripts/Set-SkipJobVariables.ps1

```diff
@@ -0,0 +1,64 @@
+. "$PSScriptRoot/../common/scripts/common.ps1"
+$RepoRoot = $RepoRoot.Path.Replace('\', '/')
+
+# Set of files that don't require build or test when changed
+$skipFiles = @(
+    'CHANGELOG.md',
+    'README.md',
+    'SUPPORT.md',
+    'TROUBLESHOOTING.md',
+    'CONTRIBUTING.md',
+    'CODE_OF_CONDUCT.md',
+    'SECURITY.md',
+    'NOTICE.txt',
+    'LICENSE'
+)
+
+# Set of directories that don't require build or test when changed
+$skipDirectories = @(
```

Comment:
I don't want to skip testing for the github or eng/common changes those do have potential impact.